Bottom: f5dc80b55c72a199cab3ec85a3eeff0800c7d994
Top:    616a9ee0b317f941b915516497a6bcb4f8efc8e9
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2016-03-13 21:50:09 +0000

Refresh of more-changes

---

diff --git a/src/polygon.h b/src/polygon.h
index 94e1a86..f5ae0b3 100644
--- a/src/polygon.h
+++ b/src/polygon.h
@@ -34,7 +34,7 @@
 
 /* Implementation constants */
 
-#define POLY_CIRC_SEGS 40 //8 //40
+#define POLY_CIRC_SEGS 20 //8 //40
 #define POLY_CIRC_SEGS_F ((float)POLY_CIRC_SEGS)
 
 #if 0
